sale_order_price_recalculation/models/sale_order.py

```diff
@@ -15,3 +19,11 @@ def action_update_names(self):
         lines = self.mapped("order_line")
         lines._compute_name()
         return True
+
+    def _recalculate_prices(self):
+        self = self.browse(self._context.get("active_ids"))
```

Comment:
The name action_update_prices is already used, but would the method name action_recompute_prices better?

Reply:
about name, the module is named recalculation, but I can change

Reply:
in this case it's intentional to prefix method name by `_` to not allow to call via xml-rpc. Are you ok ?
Then `action_` convention  is for method which are accessible by xml-rpc isn't it ?

Reply:
Then `_recalculate_prices` and `_recompute_prices` are candidates but the last could be collide other methods in other modules ? re-compute is a common word used, isn't it ?

Reply:
You are right, that action_ can be called by the API, but I do not see any problem to call this method because you could also call action_update_prices() in a loop from the API.

And action_update_prices calls _recompute_prices and I thought having **_recalculate_prices** and **_recompute_prices**  is also confising. 

When **action_recompute_prices** calls **_recompute_prices** via **action_update_prices** it's a little more consistant.

I think the best name would be **action_update_prices** but this is already used. Maybe **action_update_prices_list**

![image](https://github.com/OCA/sale-workflow/assets/1799080/137ea6e6-5661-4c8a-85e3-cd9d4e1b75dc)

But I agree there is a lot personal taste.